.azure-pipelines/generate-v1.0-sdk.yml

```diff
@@ -12,7 +12,7 @@ resources:
     type: github
     endpoint: microsoftgraph
     name: microsoftgraph/msgraph-sdk-dotnet
-    ref: dev # checkout the dev branch
+    ref: zengin/typeSummary # checkout the dev branch
```

Comment:
```suggestion
    ref: dev # checkout the dev branch
```

Reply:
this was a manual pipeline run, this shouldn't have generated a PR. sigh!

Reply:
fixed the pipeline here: https://github.com/microsoftgraph/msgraph-sdk-dotnet/pull/786 :)